src/main/java/io/kestra/plugin/notifications/opsgenie/OpsgenieTemplate.java

```diff
@@ -0,0 +1,130 @@
+package io.kestra.plugin.notifications.opsgenie;
+
+import io.kestra.core.models.annotations.PluginProperty;
+import io.kestra.core.models.tasks.VoidOutput;
+import io.kestra.core.runners.RunContext;
+import io.kestra.core.serializers.JacksonMapper;
+import io.swagger.v3.oas.annotations.media.Schema;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.ToString;
+import lombok.experimental.SuperBuilder;
+import org.apache.commons.io.Charsets;
+import org.apache.commons.io.IOUtils;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+
+@SuperBuilder
+@ToString
+@EqualsAndHashCode
+@Getter
+@NoArgsConstructor
+public abstract class OpsgenieTemplate extends OpsgenieAlert {
```

Comment:
Almost all `@Schema.title` are wrongly copy/pasted, can you update to accurate title?